sdk/keyvault/keyvault-certificates/src/generated/api/index.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export { createKeyVault, KeyVaultClientOptions, KeyVaultContext } from "./keyVaultContext";
```

Comment:
Manually converted to CJS style imports in the generated code since I didn't want to tackle the ESM migration right away